internal/transport/http2_client.go

```diff
@@ -1428,17 +1435,11 @@ func (t *http2Client) operateHeaders(frame *http2.MetaHeadersFrame) {
 	}
 
 	if statusGen == nil {
-		rawStatusCode := codes.Unknown
-		if rawStatus != "" {
-			code, err := strconv.ParseInt(rawStatus, 10, 32)
-			if err != nil {
-				se := status.New(codes.Internal, fmt.Sprintf("transport: malformed grpc-status: %v", err))
-				t.closeStream(s, se.Err(), true, http2.ErrCodeProtocol, se, nil, endStream)
-				return
-			}
-			rawStatusCode = codes.Code(uint32(code))
+		rsc := codes.Unknown
+		if rawStatusCode != nil {
+			rsc = *rawStatusCode
```

Comment:
Why not make `rawStatusCode` a non-pointer and initialize it to `codes.Unknown` instead?  Also "rsc" = "Russ Cox" :laughing: (github.com/rsc)

Reply:
I've initialized rawStatusCode as `codes.Unknown` instead. When I wrote that I was definitely thinking about Russ haha